front/components/assistant/conversation/UserMessage.tsx

```diff
@@ -25,7 +30,11 @@ export function UserMessage({
     >
       <div className="flex flex-col gap-4">
         <div>
-          <RenderMarkdown content={message.content} blinkingCursor={false} />
+          <RenderMarkdown
+            content={message.content}
+            blinkingCursor={false}
+            agentConfigurations={agentConfigurations}
```

Comment:
I understand why we have to send the agentConfigurations to the component, but that makes it tied to the Assistant > it's not anymore about rending some content, it needs to be aware of `agentConfigurations` to work. 
Could you rename the component to make it explicit and move `front/components/RenderMarkdown.tsx` to something like `front/components/assistant/RenderMessageMarkdown.tsx`?